app/views/projects/_content_item.html.erb

```diff
@@ -31,7 +31,7 @@
       &nbsp;
     </p>
     <div class='actions actions-inline'>
-      <%= button_to 'Done', mark_done_project_content_item_path(content_item.project, content_item), class: 'btn btn-primary' %>
+      <%= button_to 'Done', mark_done_project_content_item_path(content_item.project, content_item), class: 'btn btn-primary', form_class: 'js-mark-as-done', remote: true %>
```

Comment:
When this button is pressed even if no action is taken, the modal is hidden and the element is removed. 

Perhaps we could get around this with HTML5 form validation to require a field to be selected (and maybe text to be input if the second option is selected). Alternatively, hook onto this form submit event with the same style as the Done form.

![sep-21-2017 14-36-19](https://user-images.githubusercontent.com/885223/30698706-a0d9ccc4-9eda-11e7-82fd-0d65f545b1b0.gif)

Reply:
Seems this still _might_ be a problem for when the suggested tags input field is left blank after toggling the second option:

![sep-22-2017 14-23-39](https://user-images.githubusercontent.com/885223/30746671-7025c42a-9fa2-11e7-9816-2a7e4fee8868.gif)